core/src/main/java/io/questdb/std/datetime/microtime/GenericTimestampFormat.java

```diff
@@ -266,7 +266,7 @@ public void format(long micros, DateLocale locale, CharSequence timeZoneName, Ch
                     sink.put(Timestamps.getWeekOfMonth(micros));
                     break;
                 case TimestampFormatCompiler.OP_WEEK_OF_YEAR:
-                    sink.put(Timestamps.getWeekOfYear(micros));
```

Comment:
I find this confusing. Technically ISO WEEK should be serialized as 'W01' or whatever the number is. Producing String '1' is ambiguous for the reader. Also this removes existing functionality that could be used.

Reply:
@bluestreak01 Should we just create a new OP_ISO_WEEK_OF_YEAR (maybe "ww"?) and use the append0 method with that one? Also, having 'W' as a delimiter seems like it will invalidate OP_WEEK_OF_MONTH, but that one doesn't seem like it's being used as far as I know.

Reply:
yeah, i think another letter for iso week. `W` should be fine i think, so go for it!